subsys/net/lib/aws_jobs/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+zephyr_library()
+zephyr_library_sources(
+	src/aws_jobs.c
+)
+zephyr_include_directories(./include)
```

Comment:
AFAICS this is not needed since `aws_jobs.h` is in `nrf/include`.

Reply:
legacy from an older version where `aws_jobs_internal.h` was in an include folder can remove it and add it later if needed.